internal/wrappers/features/step_defs_cpp/kuzzle-sdk-steps.cpp

```diff
@@ -144,8 +145,14 @@ GIVEN("^Kuzzle Server is running$")
   ScenarioScope<KuzzleCtx> ctx;
   ctx->kuzzle_options         = KUZZLE_OPTIONS_DEFAULT;
   ctx->kuzzle_options.connect = MANUAL;
+  std::string hostname = "localhost";
```

Comment:
I don't get why we use an env. variable to use a fixed string value. Shouldn't this be something along these lines?

```
std::string hostname = "localhost";

if (std::getenv("KUZZLE_HOST") != NULL) {
  hostname = std::getenv("KUZZLE_HOST");
}
```

Or better yet, for a more explicit version which also prevents a double call to std::getenv:

```
std::string hostname = std::getenv("KUZZLE_HOST");

if (hostname == NULL) {
  hostname = "localhost";
}
```